README.md

```diff
@@ -1,12 +1,13 @@
 <!--- FIXME: Pick an emoji! --->
 # 🌻 opensource-template
 
-<!--- FIXME: Update crate and repo names here! --->
-[![Build Status](https://github.com/EmbarkStudios/template/workflows/CI/badge.svg)](https://github.com/EmbarkStudios/template/actions?workflow=CI)
-[![Crates.io](https://img.shields.io/crates/v/tame-oauth.svg)](https://crates.io/crates/tame-oauth)
-[![Docs](https://docs.rs/tame-oauth/badge.svg)](https://docs.rs/tame-oauth)
-[![Contributor Covenant](https://img.shields.io/badge/contributor%20covenant-v1.4%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
-[![Embark](https://img.shields.io/badge/embark-open%20source-blueviolet.svg)](https://embark.dev)
+<!--- FIXME: Update crate, repo and CI workflow names here! Remove any that are not relevant --->
+[![Embark](https://img.shields.io/badge/embark-open%20source-blueviolet.svg)](http://embark.games)
```

Comment:
I think we want to continue linking to embark.dev instead of the games website here, since that is where all the info around our open source will be.

Reply:
Oops. Not sure where I copied that from. I'll fix this, thanks for spotting